R/theme.r

```diff
@@ -157,6 +157,12 @@
 #'   inherits from `title`) left-aligned by default
 #' @param plot.caption caption below the plot (text appearance)
 #'   (`element_text`; inherits from `title`) right-aligned by default
+#' @param plot.tag upper-left label to identify a plot (text appearance)
+#'   (`element_text`; inherits from `title`) left-aligned by default
+#' @param plot.tag.position The position of the tag as a string ("topleft",
+#'   "top", "topright", "left", "right", "bottomleft", "bottom", "bottomright)
+#'   or a coordinate. In the former case extra space will be added to accomodate
```

Comment:
(as length-2 numeric vector)

"In former case" -> "If a string,"

Reply:
Fixed in 80d3543